packages/examples/package.json

```diff
@@ -33,8 +34,7 @@
     "demo:router": "tsx src/router.tsx",
     "demo:debug": "tsx src/debug.tsx",
     "demo:summarizer": "tsx src/summarizer.tsx",
-    "demo:zepp": "tsx src/bakeoff/zepp-health/ai-jsx.tsx",
```

Comment:
I want to keep Zepp in the repo (particularly since Larry is going to demo it on Monday). We just need to scrub the name.